src/main/java/org/ohdsi/webapi/security/PermissionService.java

```diff
@@ -285,9 +285,9 @@ public boolean hasReadAccess(CommonEntity entity) {
             try {
                 String login = this.permissionManager.getSubjectName();
                 UserSimpleAuthorizationInfo authorizationInfo = this.permissionManager.getAuthorizationInfo(login);
-                if (Objects.equals(authorizationInfo.getUserId(), entity.getCreatedBy().getId())){
-		    hasAccess = true; // the role is the one that created the artifact
-		} else {
+                // if (Objects.equals(authorizationInfo.getUserId(), entity.getCreatedBy().getId())){
+                //     hasAccess = true; // the role is the one that created the artifact - Is this for backwards compatibiliy mode? Should we write a migration instead?
+		        // } else {
```

Comment:
todo: add a line similar to this https://github.com/uc-cdis/WebAPI/pull/107/files#diff-90975f196358543a36095f13ea6d0cfaaaeb80ac8b9a3a2adfcf896ec5baa628R61 for backwards compatibility.

Reply:
done